design-documents/service-isolation.md

```diff
@@ -239,6 +239,9 @@ General principles to follow for service isolation:
 * All service operations MUST BE stateless
 * There MUST BE NO data dependencies between services.
 * Command & query responsibility segregation – storefront APIs for data immutable in storefront (catalog) should be optimized for data retrieval
+* Each request to service MUST contain correlation identifier to allow identify requests related to the same flow during debugging
```

Comment:
The same is already mentioned as "Correlation IDs must be added to internal calls to make tracing easier.".